server/sonar-web/src/main/js/apps/component-measures/components/bubbleChart/BubbleChart.js

```diff
@@ -106,6 +106,7 @@ export default class BubbleChart extends React.Component {
   }
 
   getTooltip(component) {
+    /* eslint-disable max-len */
```

Comment:
Actually you are absolutely right! `eslint-disable` disables rules until it is re-enabled by `eslint-enable`.

I've rewritten that parts of code to avoid disabling the rule.